src/cf-core/src/cf-base.less

```diff
@@ -53,7 +53,7 @@
 .u-webfont-demi() {
     font-family: @webfont-medium, Arial, sans-serif;
     font-style: normal;
-    font-weight: bold;
+    font-weight: 600;
```

Comment:
Are we sure this isn't loading the local version of `600`? Have we tested this on a Win machine?

Reply:
Pretty sure it can't be using the local font, because we're using `@font-face` to declare what the source of the font family `'Avenir Next'` is, and we aren't including local sources there. There's a Windows Firefox (looks great) and Windows Edge (looks like it's just using Medium) screenshot up in the OP, though ☝️